src/main/java/com/minecolonies/coremod/items/ItemSupplyChestDeployer.java

```diff
@@ -89,43 +89,12 @@ public ItemSupplyChestDeployer()
         setMaxStackSize(1);
     }
 
-    @NotNull
-    @Override
-    public EnumActionResult onItemUse(
-                                       final EntityPlayer playerIn,
-                                       final World worldIn,
-                                       final BlockPos pos,
-                                       final EnumHand hand,
-                                       final EnumFacing facing,
-                                       final float hitX,
-                                       final float hitY,
-                                       final float hitZ)
-    {
-        ItemStack stack = playerIn.getHeldItem(hand);
-        if (worldIn.isRemote || stack.getCount() == 0 || !isFirstPlacing(playerIn))
-        {
-            return EnumActionResult.FAIL;
-        }
-
-        @NotNull final EnumFacing enumfacing = canShipBePlaced(worldIn, pos);
-        if (enumfacing != EnumFacing.DOWN)
-        {
-            spawnShip(worldIn, pos, enumfacing);
-            stack.setCount(stack.getCount() - 1);
-
-            playerIn.addStat(ModAchievements.achievementGetSupply);
-
-            return EnumActionResult.SUCCESS;
-        }
-        LanguageHandler.sendPlayerMessage(playerIn, "item.supplyChestDeployer.invalid");
-        return EnumActionResult.FAIL;
-    }
-
     /**
      * Checks if the player already placed a supply chest.
      *
      * @param player the player.
-     * @return boolean, returns true when player hasn't placed before, or when infinite placing is on.
+     * @return boolean, returns true when player hasn't placed before, or when
+     * infinite placing is on.
      */
     private static boolean isFirstPlacing(@NotNull final EntityPlayer player)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "isFirstPlacing" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)